src/APIFunctions/2DPrinting.js

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 import { PrintApiResponse } from './ApiResponses';
-import { RPC_API_URL } from '../config/config';
+const { RPC_API_URL } = require('../config/config.json');
```

Comment:
for importing stuff in `src/` use the convention like the line above:

```suggestion
import { RPC_API_URL } from '../config/config.json';
```
please fix for all other files in `src`